tools/docker/images/builder/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:18.04
+
+ENV https_proxy=http://proxy-iil.intel.com:911
+ENV http_proxy=http://proxy-iil.intel.com:911
+ENV no_proxy=localhost,127.0.0.0/8,192.168.0.0/16,.intel.com,.lantiq.com
+ENV ftp_proxy=http://proxy-iil.intel.com:911
+
+# Update Software repository
+RUN apt-get update && apt-get install -y \
+	curl \
+	gcc \
+	cmake \
+	ninja-build \
+	binutils \
+	git \
+	autoconf \
+	autogen \
+	libtool \
+	pkg-config \
+    	libreadline-dev \
+	libncurses-dev \
+	libssl-dev \
+	libjson-c-dev \
+	libnl-genl-3-dev \
+	libzmq3-dev \
+    	python \
+	python-yaml \
+	repo \
+	vim \
+    && rm -rf /var/lib/apt/lists/*
+
+
+# Set up git and google repo
+WORKDIR /root/prplMesh
+RUN git config --global user.email prplMesh@prpl.com && \
```

Comment:
Did you intentionally use a non-existent mail address?

Maybe better use prplMesh@lists.prplfoundation.org?